src/lib/dom.generated.d.ts

```diff
@@ -4928,8 +4943,10 @@ interface EventSource extends EventTarget {
     readonly CONNECTING: number;
     readonly OPEN: number;
     addEventListener<K extends keyof EventSourceEventMap>(type: K, listener: (this: EventSource, ev: EventSourceEventMap[K]) => any, options?: boolean | AddEventListenerOptions): void;
+    addEventListener(type: string, listener: (this: EventSource, event: MessageEvent) => any, options?: boolean | AddEventListenerOptions): void;
```

Comment:
What's the rationale for these new overloads?

Reply:
https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/736 with https://github.com/microsoft/TypeScript-DOM-lib-generator/pull/1182 - basically default to message event if we don't know the event name